deployment/GP02HYPM_Deploy.csv

```diff
@@ -1,99 +1,99 @@
+SR1710,,SR1811,,GP02HYPM-WFP03-04-CTDPFL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.1156,-144.9176,,4063,4230,
+SR1811,,SKQ201920S,,GP02HYPM-MPM01-02-ZPLSGA009,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,,CGINS-ZPLSGA-55093,50.06856,-144.79826,,150,4222,upward-looking
+SR1811,,SKQ201920S,,GP02HYPM-MPM01-02-ZPLSGA010,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,,CGINS-ZPLSGA-55092,50.06856,-144.79826,,150,4222,downward-looking
+SR1811,,SKQ201920S,,GP02HYPM-RIM01-00-SIOENG000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,,CGCON-SIOCON-10118,50.06856,-144.79826,,4073,4222,
+SR1811,,SKQ201920S,,GP02HYPM-RIM01-02-CTDMOG039,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,,CGINS-CTDMOG-13622,50.06856,-144.79826,,160,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-00-WFPENG000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGCON-GWFPGX-10402,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-01-FLORDL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGINS-FLORDL-03283,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-03-DOSTAL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGINS-DOSTAL-01480,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-04-CTDPFL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-05-VEL3DL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-00-WFPENG000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-01-FLORDL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-03-DOSTAL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-04-CTDPFL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-05-VEL3DL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.06856,-144.79826,,4063,4222,

```

Comment:
Deployment 6: WFP03 and SIOENG stopDateTime should be 2019-09-26T22:09 (per mooring recovery log).

Reply:
done